### PR TITLE
Various small tweaks

### PR DESCRIPTION
**Pull request by Sybrand**
# Overview
Addressing various small issues that bothered me while switching over to my new computer.

# Changes
- Added .env file in root (needed for vscode tests to work correctly), which meant not having a blanket .env ignore rule.
- Adding launch.json and settings.json configuration for vscode, which meant not having blank .vscode ignore rule, and adding poetry.toml file to force common location for python virtual env.
- Updated and added info to README files.
- Adding logging exception in audits. (otherwise, if db is configured incorrectly, it will fail without logging anything)
- Added a default failback for missing static folder. That's all garbage code that needs to be made redundant anyway by serving up the static content elsewhere. (Without this, an incorrectly configured static folder causes unit tests to fail, which in turn result in unit test discovery failure in vscode)
- Added example .env 
- Made some changes to s3-backup
# Test Links:
[Percentile Calculator](https://wps-pr-1507.apps.silver.devops.gov.bc.ca/)
[MoreCast](https://wps-pr-1507.apps.silver.devops.gov.bc.ca/morecast)
[C-Haines](https://wps-pr-1507.apps.silver.devops.gov.bc.ca/c-haines)
[FireBat](https://wps-pr-1507.apps.silver.devops.gov.bc.ca/fire-behaviour-calculator)
[FireBat bookmark](https://wps-pr-1507.apps.silver.devops.gov.bc.ca/fire-behaviour-calculator?s=266&f=c5&c=NaN&w=20,s=286&f=c7&c=NaN&w=16,s=1055&f=c7&c=NaN&w=NaN,s=305&f=c7&c=NaN&w=NaN,s=344&f=c5&c=NaN&w=NaN,s=346&f=c7&c=NaN&w=NaN,s=328&f=c7&c=NaN&w=NaN,s=1399&f=c7&c=NaN&w=NaN,s=334&f=c7&c=NaN&w=NaN,s=1082&f=c3&c=NaN&w=NaN,s=388&f=c7&c=NaN&w=NaN,s=309&f=c7&c=NaN&w=16,s=306&f=c7&c=NaN&w=NaN,s=1029&f=c7&c=NaN&w=NaN,s=298&f=c7&c=NaN&w=NaN,s=1108&f=c5&c=NaN&w=NaN,s=836&f=c7&c=NaN&w=NaN#state=2ec784ca-c46a-49d0-b2b3-1cf32a9015a2&session_state=7d9447c8-db66-4661-b4cb-03d2ac0d1d8f&code=32292df4-2bdf-4f90-a4a8-c8dbcda682a9.7d9447c8-db66-4661-b4cb-03d2ac0d1d8f.2b63f390-f3dc-43ae-89f2-016453863476)
[Fire Behaviour Advisory](https://wps-pr-1507.apps.silver.devops.gov.bc.ca/fire-behaviour-advisory)
[HFI Calculator](https://wps-pr-1507.apps.silver.devops.gov.bc.ca/hfi-calculator)

### CODE REVIEW

**Review by conbrad — APPROVED** [review]
Ship it! 🚢

File: api/app/db/crud/api_access_audits.py [hunk 1]
```diff
@@ -14,9 +14,13 @@ def create_api_access_audit_log(
...
+    try:
+        with app.db.database.get_write_session_scope() as session:
+            now = get_utc_now()
+            audit_log = APIAccessAudit(create_user=username, path=path, success=success,
+                                       create_timestamp=now)
+            session.add(audit_log)
+            session.commit()
+    except Exception as exception:
+        logger.error(exception, exc_info=True)
+        raise
```
Comment (conbrad) [inline api/app/db/crud/api_access_audits.py:26]: Codecov patch check wants a test for this case, kind of annoying but easy to add I think

File: .vscode/launch.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "api",
+            "type": "python",
```
Comment (conbrad) [inline .vscode/launch.json:9]: I had to uninstall and reinstall the `python` extension for this to work, other than that, no issues.

File: api/app/db/crud/api_access_audits.py (hunk 1 above)
Comment (Sybrand) [reply api/app/db/crud/api_access_audits.py:26]: Oh dear me - except that I forgot how to write tests, and that took me forever. Hopefully we're good now.

**Pull request merged**